runtimes/quick-start.md

```diff
@@ -8,74 +8,102 @@ description: >-
 
 {% tabs %}
 {% tab title="Web" %}
+
 ## 1. Add the Rive library
 
 Add a script tag to a web page:
 
+```html
```

Comment:
I really only touched the `Web` tab content, most of the stuff beyond this is auto-formatting from `Prettier`.